src/main/java/core/basesyntax/strategy/handler/BalanceOperationHandler.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy.handler;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class BalanceOperationHandler implements OperationHandler {
+
```

Comment:
Empty line

Reply:
this empty line separates imports and class. we need it

Reply:
Still empty line